src/main/java/com/thealgorithms/maths/FindMaxRecursion.java

```diff
@@ -49,6 +30,6 @@ public static int max(int[] array, int low, int high) {
      * @return max value of {@code array}
      */
     public static int max(int[] array) {
```

Comment:
```suggestion
    public static int max(final int[] array) {
```

Please also mark the arguments of the _other_ `max` method as `final`.